ocular-dev-tools.config.js

```diff
@@ -0,0 +1,30 @@
+const {resolve} = require('path');
+
+module.exports = {
+  babel: {
+    plugins: [['transform-builtin-extend', {globals: ['Array']}]]
+  },
+
+  lint: {
+    paths: ['src', 'test', 'examples'],
+    extensions: ['js']
+  },
+
+  aliases: {
+    'math.gl/test': resolve(__dirname, './test')
+  },
+
+  entry: {
+    'test-node': 'test/index.js',
+    size: {
+      all: 'test/size/all.js',
```

Comment:
At some point in the future, we could just scan the top level export for symbols and auto generate these.